dspace-api/src/main/java/org/dspace/qaevent/QATopic.java

```diff
@@ -8,18 +8,24 @@
 package org.dspace.qaevent;
 
 import java.util.Date;
+import java.util.UUID;
 
 /**
  * This model class represent the quality assurance broker topic concept. A
  * topic represents a type of event and is therefore used to group events.
  * 
  * @author Andrea Bollini (andrea.bollini at 4science.it)
- *
  */
 public class QATopic {
+
     private String key;
-    private long totalEvents;
+    /**
+     * if the QASource stats (see next attributes) are related to a specific target
+     */
+    private UUID focus;
```

Comment:
yes, it is the same concept used in qasource applied to the QATopic and the javadoc contains a typo mentioning QASource.
We are going to update it as follow

```The focus attributes specify if the QATopic object is describing the status of a specific quality assurance topic for the whole repository (focus = null) or for a specific DSpaceObject (focus = uuid of the DSpaceObject). This would mostly affect the totalEvents attribute below```